packages/react-core/src/components/BackToTop/BackToTop.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/BackToTop/back-to-top';
+import { css } from '@patternfly/react-styles';
+import AngleUpIcon from '@patternfly/react-icons/dist/esm/icons/angle-up-icon';
+import { canUseDOM } from '../../helpers/util';
+import { Button } from '../Button';
+
+interface BackToTopProps extends React.DetailedHTMLProps<React.HTMLProps<HTMLDivElement>, HTMLDivElement> {
+  /** Additional classes added to the back to top. */
+  className?: string;
+  /** Title to appear in back to top button, defaults to 'Back to top'. */
```

Comment:
It is not necessary to list the default sine there is a column for that in the docs.